src/examples/ex5f.f

```diff
@@ -40,8 +40,12 @@ program ex5f
 
       integer    MAX_LOCAL_SIZE
       integer    HYPRE_PARCSR
+      integer    HYPRE_MEMORY_DEVICE
+      integer    HYPRE_EXEC_DEVICE
 
       parameter  (MAX_LOCAL_SIZE=123000)
+      parameter  (HYPRE_MEMORY_DEVICE=1)
```

Comment:
Why is this both here and in the `HYPREf.h` file?

Reply:
I don't see `HYPREf.h` is included. I might be wrong.

Reply:
Oh, then we need to add the include.  That's why it's there.  It's in `ex12f.f`, but not this one.